content/en/containers/amazon_ecs/_index.md

```diff
@@ -363,7 +411,7 @@ Need help? Contact [Datadog support][11].
 [1]: https://docs.datadoghq.com/integrations/faq/agent-5-amazon-ecs/
 [2]: https://docs.datadoghq.com/agent/docker/integrations/?tab=docker
 [3]: https://docs.datadoghq.com/integrations/ecs_fargate/
```

Comment:
I don't think these are referenced anywhere anymore? Also, [11] is not referenced.
```suggestion
```